tests/exploits/cve_2014_0780_test.py

```diff
@@ -60,3 +59,4 @@ def testCVE20140780_whenNotVulnerable_noFindingsReported(
     vulnerabilities = exploit_instance.check(target)
 
     assert len(vulnerabilities) == 0
+    assert exploit_instance.session.max_redirects == 2
```

Comment:
These are not needed and also not useful. You are checking a value you are setting, not a behavior you are monitoring.